src/Symfony/Component/Finder/Tests/Iterator/VcsIgnoredFilterIteratorTest.php

```diff
@@ -51,7 +51,12 @@ public function testAccept(array $gitIgnoreFiles, array $otherFileNames, array $
 
         $inner = new InnerNameIterator($otherFileNames);
 
-        $iterator = new VcsIgnoredFilterIterator($inner, $this->tmpDir);
+        if ('' !== $baseDir) {
+            $baseDir = "{$this->tmpDir}/{$baseDir}";
+        } else {
+            $baseDir = $this->tmpDir;
+        }
```

Comment:
```suggestion
        $baseDir = $this->tmpDir;
        if ('' !== $baseDir) {
            $baseDir .= '/'.$baseDir;
        }
```

Reply:
Would need to use another variable name then to not overwrite  it in line 54. But since you approved already I guess it's fine?

Reply:
ah indeed, suggestion updated

Reply:
Done